Azure/communication-ui-library-ios
# [Bugfix] Fixed the one UITest failure reported in app center about setting text in acs text field

## Purpose

* Fixed the one UITest failure reported in app center about setting text in acs text field , that in some ios15 devices, app is not able to perform needed typetext nor past text into a textfield, which eventually lead to test failure. This PR has a work around method to be able to set acs token field so it would pass the UI test in app center.

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
Perform a UItest in Xcode, and make sure the test runs correctly (100% pass rate)
Check the UI test result in app center link will be provided later.
```

## What to Check
Verify that the following are valid
* ...

## Other Information
